docs/configuration.md

```diff
@@ -300,6 +300,13 @@ If enabled, a system message will be posted into any open threads if the user jo
 **Default:** `on`  
 If enabled, a system message will be posted into any open threads if the user leaves a main server
 
+#### overrideRoleNameDisplay
+**Default:** `None`  
+Role name to display in all replies. This completely overrides normal role selection, all replies will contain the string entered. For example;
+```ini
```

Comment:
Full code block comments should be reserved for more elaborate config options.
I would change this to use regular inline highlights using ticks, like here for example:

> #### reactOnSeenEmoji
> **Default:** `📨`  
> The emoji that the bot will react with when it sees a message.  Requires `reactOnSeen` to be enabled.  
> Must be pasted in the config file as the Emoji representation and not as a unicode codepoint. Use `emojiName:emojiID` for custom emoji.

Reply:
Done